cmd/kpod/common.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var needToShutdownStore = false
+
+func getStore(c *cli.Context) (storage.Store, error) {
+	options := storage.DefaultStoreOptions
+	if c.GlobalIsSet("root") || c.GlobalIsSet("runroot") {
```

Comment:
man buildah shows
```
       --root value

       Storage root dir (default: "/var/lib/containers/storage")

       --runroot value

       Storage state dir (default: "/var/run/containers/storage")
```
Looks like these allow you to specify alternative root directories.  I don't think we should expose these to the users.